### DISCUSSION TITLE
Performance by category in rel_component

### DISCUSSION THREAD

**Discussion by rjaas** [explosion/spaCy#9739] [Help: Coding & Implementations]
I have successfully trained a relation extractor according to [the tutorial]( https://www.youtube.com/watch?v=8HL-Ap5_Axo). However, when I trained a NER model, _meta.json_ reported performance for each Named Entity category (e.g Person, Time), whilst the custom relation extractor does not.

How can I get precision, recall and f-score per relationship category when training the custom relation extractor?

**Comment by polm** [comment]
The REL component doesn't have an option for that, you'd need to customize the scoring function.
